docs/libfwupd/libfwupd-docs.xml

```diff
@@ -401,6 +401,17 @@ fu_plugin_update_reload (FuPlugin *plugin, FuDevice *device, GError **error)
         </para>
       </section>
 
+      <section>
+        <title>Debugging a Plugin</title>
+        <para>
+          If the fwupd daemon is started with  <code>--plugin-verbose=unifying</code>
+          then the environment variable <code>FWUPD_UNIFYING_VERBOSE</code> is
+          set process-wide.
```

Comment:
Right now this reads as all plugin verbosity can be set by specifically --plugin-verbose=unifying and all plugins need to look for FWUPD_UNIFYING_VERBOSE.
Can you change this to --plugin-verbose=$plugin then the environment variable FWUPD_$PLUGIN_VERBOSE is set for the initial description and then give the unifying example afterward?  I think it will be clearer that all plugins can support this.

Reply:
Fixed.